docs/UC_CV_3ElectrodeCell.md

```diff
@@ -0,0 +1,40 @@
+Use Case: Cyclic Voltammetry on the JUAMI Potentiostat
+Intended User: Undergraduate Researcher at a University in Africa with Introductory Knowledge of Electrochemistry and Cyclic Voltammetry. The instructions in this UC assume that the student already has an electrochemical cell set up and ready for testing (working electrode, counter electrode, and possibly reference electrode are all in a proper beaker).
+As a test case - Mike will perform these instructions as someone with knowledge of electrochemistry but little knowledge of Python.
+
+Experimental Setup:
+
+1. Mike plugs the USB cable into the computer and the potentiostat
+1. Mike attaches the red wire to the working electrode (material of interest)
+1. Mike attaches the black clip to the counter electrode
+1. Mike attaches the white clip to the reference electrode if he is using one, or he attaches it to the counter electrode for a two electrode test
+
+Program Setup:
+1. Mike opens the Pytentiostat script (the script is incomplete and requires filling in of user-defined information)
+1. Mike types a file name for the sample being tested next to 'file name' on line # (file will contain the voltage, current, and time information)
+1. Mike inputs Initial Potential on line # (V vs. open circuit voltage (Eocv))
+1. Mike inputs first turnover potential (V vs. Ref) on line #
+1. Mike inputs second turnover potential (V) on line #
+1. Mike inputs sweep rate (mV/s) on line #
+1. Mike inputs number of times the cycle should repeat on line #
+1. Mike inputs final potential on line # (typically 0 V vs. Eocv same as initial potential)
+1. Mike types the lower limiting safety potential on line # (the program will automatically turn off if potential exits this range)
+1. Mike types the upper limiting safety potential on line #
```

Comment:
These are all excellent.  Just the right level of detail, and we can reuse this in the "Getting started" documentation, modulo what I said above about new users knowing what values to type in there